libs/gateway-api/config.jsonnet

```
@@ -1,13 +1,22 @@
 local config = import 'jsonnet/config.jsonnet';
 
+local versions = [
+    {output: '0.7', version: '0.7.1'},
+    {output: '0.8', version: '0.8.1'},
+    {output: '1.0', version: '1.0.0'},
+];
+
 config.new(
   name='gateway-api',
   specs=[
     {
-      output: 'v0.7.1',
+      output: v.output,
       prefix: '^io\\.k8s\\.networking\\.gateway\\..*',
-	  crds: ['https://github.com/kubernetes-sigs/gateway-api/releases/download/v0.7.1/standard-install.yaml'],
+	    crds: [
+        'https://github.com/kubernetes-sigs/gateway-api/releases/download/v$(version)s/standard-install.yaml' % { version: v.version }
```

```suggestion
        'https://github.com/kubernetes-sigs/gateway-api/releases/download/v%(version)s/standard-install.yaml' % { version: v.version }
```

I think this should fix it.

---

I did not see that. Thanks! 🙏